frontend/src/fixtures/courseOverTimeSearchFixtures.js

```diff
@@ -0,0 +1,28 @@
+const coursesOverTimeSearchFixtures = {
+    twoCourses:
+      [
+        {
+        "startQuarter": "20211",
+        "endQuarter": "20214",
+        "subjectArea": "CMPSC ",
+        "courseNumber": "8  "
+      },
+      {
+        "startQuarter": "20201",
+        "endQuarter": "20204",
+        "subjectArea": "CMPSC ",
+        "courseNumber": "CMPSC     16  "
+      }
+    ],
+    oneCourse:
+      [
+        {
+          "startQuarter": "20084",
+          "endQuarter": "20214",
+          "subjectArea": "CMPSC ",
+          "courseNumber": "8  "
+        }
+    ]
+  }
+  
```

Comment:
Are these used anywhere?  I couldn't find any place they were.

We typically create fixtures when we are mocking the JSON that is returned by a backend endpoint, but I don't see any backend endpoint that would be returning this; instead, these look like the parameters that we'd be passing to the backend.

If I'm missing something, let me know, but I'm wondering if we can delete this.

Reply:
No they aren't. I deleted the file in my new commit.